<ui version="4.0" >
 <author>Matthias Kretz &lt;kretz@kde.org></author>
 <class>BackendSelection</class>
 <widget class="QWidget" name="BackendSelection" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>362</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" colspan="3" >
      <widget class="QListWidget" name="m_select" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>A list of Phonon Backends found on your system.  The order here determines the order Phonon will use them in.</string>
       </property>
       <property name="whatsThis" >
        <string>A list of Phonon Backends found on your system.  The order here determines the order Phonon will use them in.</string>
       </property>
       <property name="dragEnabled" >
        <bool>true</bool>
       </property>
       <property name="dragDropMode" >
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QToolButton" name="m_up" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Prefer</string>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QToolButton" name="m_down" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Defer</string>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>81</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="m_icon" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>0</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_name" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="alignment" >
           <set>Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line1" >
        <property name="frameShape" >
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_comment" >
        <property name="text" >
         <string/>
        </property>
        <property name="alignment" >
         <set>Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_website" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_version" >
        <property name="text" >
         <string/>
        </property>
        <property name="alignment" >
         <set>Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="2" row="0" column="1" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="page" />
     <widget class="QWidget" name="page_2" />
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
